pkg/flowaggregator/flowaggregator.go

```diff
@@ -294,6 +299,31 @@ func (fa *flowAggregator) InitAggregationProcess() error {
 	return err
 }
 
+func (fa *flowAggregator) createAndSendTemplate(isRecordIPv6, isOriginExporterIPv6 bool) (uint16, error) {
+	templateID := fa.exportingProcess.NewTemplateID()
+	// If Pod IPs (source and destination IP) in the flow record belong to IPv4 Family and
+	// original exporter IP belongs to IPv6 family, we will send template with ID templateIDv4Expv6,
+	// which has sourceIPv4Address, destinationIPv4Address and originalExporterIPv6Address.
+	// Same applies to other combinations.
+	recordIPFamily := "IPv4"
+	exporterIPFamily := "IPv4"
+	if isRecordIPv6 {
+		recordIPFamily = "IPv6"
+	}
+	if isOriginExporterIPv6 {
+		exporterIPFamily = "IPv6"
+	}
+	bytesSent, err := fa.sendTemplateSet(isRecordIPv6, isOriginExporterIPv6)
+	if err != nil {
+		fa.exportingProcess.CloseConnToCollector()
+		fa.exportingProcess = nil
+		fa.set.ResetSet()
+		return 0, fmt.Errorf("sending %s template set with %s original exporter ip failed, err: %v", recordIPFamily, exporterIPFamily, err)
+	}
+	klog.V(2).InfoS("Initialized exporting process and sent %d bytes size of %s template set with %s original exporter ip", bytesSent, recordIPFamily, exporterIPFamily)
```

Comment:
Sorry I missed this earlier. This is not how the structure logging functions should be used. Please refer to https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/migration-to-structured-logging.md#structured-logging-in-kubernetes.